web_src/js/index.js

```diff
@@ -352,11 +352,10 @@ function initImagePaste(target) {
     const field = this;
     field.addEventListener('paste', (event) => {
       retrieveImageFromClipboardAsBlob(event, (img) => {
-        const name = img.name.substr(0, img.name.lastIndexOf('.'));
-        insertAtCursor(field, `![${name}]()`);
+        insertAtCursor(field, `![image]()`);
```

Comment:
```suggestion
        insertAtCursor(field, `![Uploading image ...]()`);
```

Reply:
Not sure if this is helping any, I prefer the text to not change after upload.